testsuite/integration-arquillian/tests/base/src/test/resources/META-INF/keycloak-server.json

```diff
@@ -195,7 +195,12 @@
     "vault": {
         "files-plaintext": {
             "dir": "target/dependency/vault",
-            "enabled": "${keycloak.vault.plaintext.provider.enabled:false}"
+            "enabled": "${keycloak.vault.files-plaintext.provider.enabled:false}"
+        },
+        "elytron-cs-keystore": {
+            "location": "target/dependency/vault/credential-store.jceks",
```

Comment:
I'm not really sure. The reason for this to default to JCEKS is because this is also the default used by the elytron tools when creating a credential store.

Reply:
An alternative would be to create a test keystore using the PKCS#11 format - elytron supports that for sure for FIPS

Reply:
Went with a PKCS12 format that seems to work well with Elytron and the jvm matrix.